apps/python-service/routers/ai.py

```
@@ -0,0 +1,67 @@
+        # Create the LangChain chain
+        chain = prompt | structured_llm
+        
+        # Invoke the chain
+        result = await chain.ainvoke({
+            "schema": str(request.schema_fields),
+            "user_prompt": request.prompt
+        })
+        
+        return result
+        
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)